### PR TITLE
Cisco ucs fi 6332

### PR DESCRIPTION
**Pull request by christopherkettell**
Created Cisco ucs fi 6332

### CODE REVIEW

**Comment by kaechele** [comment]
A few observations:

1. Your pull request contains a lot of commits and changes that do not relate to the UCS FI 6332. My personal recommendation is: You should try to get the number of commits per branch that you want pulled down to one per definition file / device family.
2. The "human friendly" model description for the UCS FI 6332 is "Cisco UCS 6332 Fabric Interconnect"
3. It is not necessary to include `mgmt_only` for all interfaces if the value is `false` as it is the default anyway. I get that it's more explicit but it makes the file less readable.
4. Try to be a bit more accurate and consistent with capitalization and commit messages. This will give your pull request a more uniform and clean look and will make it easier for the maintainers to grasp what you are proposing.

**Comment by christopherkettell** [comment]
@kaechele - thank you for the feedback, taken on board. I am getting a lot of value from Netbox so wnt to be able to give some back.

File: device-types/APC/AP8853.yaml [hunk 1]
```diff
@@ -0,0 +1,184 @@
+manufacturer: APC
```
Comment (DanSheps) [inline device-types/APC/AP8853.yaml:1]: This is not a UCS device type, please break it out

File: device-types/Cisco/DN2-HW-APL.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+manufacturer: Cisco
+model: DN2-HW-APL
```
Comment (DanSheps) [inline device-types/Cisco/DN2-HW-APL.yaml:2]: This is not a UCS FI device type, please break it out

**Comment by DanSheps** [comment]
As there has been no activity on this in awhile, I am going to close this PR.  You are free to re-open this PR once you have made the changes requested.

**Pull request closed without merging**